### DISCUSSION TITLE
Wouldn't it make sense to allow issues without a minimal reproduction?

### DISCUSSION THREAD

**Discussion by capaj** [vitest-dev/vitest#2464] [Q&A] [answered]
As per title.
Sometimes a bug is not easily reproduceable. In that case should we use discussions instead of reporting an issue?
If that is so, could we update the template for bug with this info to save time for people who want to report an issue without a repro?

**Comment by capaj** [comment]
The current issue template states that issue without a repro will be closed, not deleted. Yet my issue for 0.25.5 was deleted, not closed. Can you comment why @sheremet-va ?

**Reply by sheremet-va** [reply]
Your issue wasn't closed nor deleted: https://github.com/vitest-dev/vitest/issues/2451

I don't think it's possible to delete issues 🤔

**Reply by capaj** [reply]
oh sorry, I could not find it at all and got really confused. Thanks for the link.

**Comment by sheremet-va** [comment]
Just gonna leave this here:

- https://gist.github.com/Rich-Harris/88c5fc2ac6dc941b22e7996af05d70ff
- https://antfu.me/posts/why-reproductions-are-required

**Comment by capaj** [comment, marked as answer]
I think the current process with adding label and closing in 3 days is :ok_hand:

I thought the process is to delete, which would be bad
